Add a copy button to code snippet

Akasurde opened the pull request:
##### SUMMARY

Fixes: #75

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE

- Docs Pull Request

Akasurde committed:
Add a copy button to code snippet

Fixes: #75

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

oraNod reviewed (approved):
:tada: Love it. Thanks @Akasurde

gotmax23 committed:
ci: add sphinx-copybutton to requirements-relaxed.txt

Both requirements-relaxed.txt and requirements.txt need to be refreshed
when adding new dependencies.

gotmax23 commented:
@samccann, @oraNod, do we want to backport this at all or leave it on devel for now?

oraNod commented:
@gotmax23 Good question. I'll leave it to @samccann to decide whether to backport. I feel like it should be OK but maybe we should give it more time on devel for evaluation.

gotmax23 commented:
Just so we're on the same page, here is an example of a codeblock: https://ansible--710.org.readthedocs.build/projects/ansible-core/en/710/inventory_guide/intro_inventory.html#inventory-basics-formats-hosts-and-groups

If you hover over it, there's a copy button in the top right that automatically copies the code to a user's clipboard.

![image](https://github.com/ansible/ansible-documentation/assets/9920591/4aa9b039-5477-430c-9c7a-3bce0ba501d5)

oraNod commented:
Yep. The only thing I would have any concern about is if that clipboard thing shows up somewhere it really shouldn't or causes some kind of unexpected behaviour. It will be available on codeblocks that aren't actually meant to be copy/pasted but that doesn't seem like an issue to me.

samccann commented:
Happy to have it backported at least to 2.16 (what will be the new latest). I don't have a problem with it going further back as well. Will leave it to y'all to decide :-)

gotmax23 commented:
> Happy to have it backported at least to 2.16 (what will be the new latest). I don't have a problem with it going further back as well. Will leave it to y'all to decide :-)

Cool, thanks. I marked this with `backport-2.16`, but we can probably deploy to devel and let the backport PR wait for a couple days.

Akasurde commented:
@samccann @gotmax23 @oraNod Thanks for the review and merge.
